deploy.cmd

```diff
@@ -0,0 +1,29 @@
+
+@echo off
+
+"tools\nuget\nuget.exe" "install" "FAKE.Core" "-OutputDirectory" "tools" "-ExcludeVersion" "-version" "4.4.2"
+
+:Build
+cls
+
+SET TARGET="CreatePackages"
+SET GITOWNER="octokit"
+SET GITPASSWORD=""
```

Comment:
Apologies, I wasn't clear.

I was suggesting this be renamed from `GITPASSWORD` to `GITHUBPASSWORD`.